ocw/lib/k8s.py

```diff
@@ -16,3 +16,21 @@ def clean_jobs(provider: Provider, client: BatchV1Api, cluster_name: str):
             else:
                 provider.log_info(f"Skip deleting from {cluster_name} the job {job.metadata.name} " +
                                   f"with age {age} (days)")
+
+
+def clean_namespaces(provider: Provider, client: CoreV1Api, cluster_name: str):
+    now = datetime.now(timezone.utc)
+    # Retrieve the list of all namespaces
+    namespaces = client.list_namespace(watch=False)
+
+    for ns in namespaces.items:
+        age = (now - ns.metadata.creation_timestamp).days
+        if (ns.metadata.name.startswith('helm-test') and age > 7):
+            # Delete the namespace
+            if not provider.dry_run:
```

Comment:
if you will swap it . it will be more readable

Reply:
Like
```
if dry run {
  skip deleting
} else {
  delete
}
```
?